nixos/modules/security/pam.nix

```diff
@@ -212,6 +212,15 @@ let
         '';
       };
 
+      enableKwallet = mkOption {
+        default = false;
+        type = types.bool;
+        description = ''
+          Enable pam_kwallet to automatically unlock user KDE Wallets
+          upon login (if their wallet password matches their login password)
```

Comment:
What happens if this is enabled, but my wallet password does not match my login password?